Libraries/Core/ReactNativeVersion.js

```diff
@@ -11,7 +11,7 @@
 
 exports.version = {
   major: 0,
-  minor: 66,
+  minor: 0,
```

Comment:
Think it'd be worth adding a // macOS comment here (and elsewhere) so we can be more careful about merging stable branches into main? Especially if it's a diff with upstream anyway

Reply:
Like a "main branch" specific comment, I mean

Reply:
I'm actually removing a diff from upstream here. We ended up with these because we merged RN 0.6x-stable into RN-macOS main, rather than RN main into RN-macOS main.

https://github.com/facebook/react-native/blob/main/ReactCommon/cxxreact/ReactNativeVersion.h

Reply:
Oh, you're saying to leave a comment for future mergers. My bad!

Not sure if I can do that in the gradle files.

Reply:
Looking at the top of `ReactNativeVersion.(h|js|java)` it looks like these files are generated:
```
 @generated by scripts/bump-oss-version.js
 ```
 
 If I add a comment, I think there's a good chance i'll cause diffs / get overwritten when that script is run by upstream again. I also can't add a comment to `package.json` files, which was the most important one since that's what the CircleCI test references. 
 
 Do you think it'd still be worth adding a comment knowing these are generated files?

Reply:
If it's generated then changing the version number by hand like in this PR would just get wiped too?